docs/agents/sematext-agent/autodisco/removing-stale-resources.md

```diff
@@ -1,5 +1,7 @@
 title: Removing stale resources
 
+description:  Removing AA containers/networks after uninstalling the agent
```

Comment:
Can you replace AA with App Agent, for it to make more sense? Many people don't know what AA is. Also, please check the whole document here and change AA to be App Agent.

Reply:
description:  Removing AA containers/networks after uninstalling the agent
=>
description:  Removing App Agent containers and networks after uninstalling the Sematext Agent.